resources/views/components/form/tags.blade.php

```diff
@@ -40,7 +40,15 @@
             }
         }
     }
+} elseif (empty($model)) {
+    $tags = \App\Models\Tag::autoApplied()->get(); 
```

Comment:
For performance reason, we need to add `->with('entity')` before `->get()`, so that getting the entity object asosciated with each tag is only a single database. Why do we need to check the tag's entity? It's (always) because of the permission engine doing weird things :P